### ISSUE TITLE
How to move object or rotate the object interactively ?

### ISSUE THREAD

**Issue by OhmPuchiss** [marcomusy/vedo#1094]
I want to rotate and translate an object interactively.

**Comment by OhmPuchiss** [comment]
I figure it out. Just pressing button a. I have a second question how do we scale an object (i.e. similar to blender by pressing s and moving left or right).

**Comment by OhmPuchiss** [comment]
Dear Marco,
Is there anyway we can access the axis of the object and its orientation ? 
I want to do thing similar to this post. However, I would like to define my own transformation matrix, button and much more. Therefore, is there anyway can I get the information about orientation and axis of the object ?
![image](https://github.com/marcomusy/vedo/assets/71385431/38114f1d-3fd1-4a07-910e-8b61b8d7b79b)

**Comment by marcomusy** [comment]
Sure you can access it by
```
myobj.transform
```
and 

```
myobj.transform.matrix
```

To apply a trasformation to a specific object [you can use](https://vedo.embl.es/autodocs/content/vedo/vedo/core.html#PointAlgorithms.apply_transform)
```
myobj.apply_transform(...)
```

**Comment by OhmPuchiss** [comment]
Hey Marco,
Is there any way to make the line rotate or move together with the object (the purple one) when using mouse to move or rotate the object ?
![image](https://github.com/marcomusy/vedo/assets/71385431/bb75167e-7d6b-43e5-85d5-7aadeaa346c8)
![image](https://github.com/marcomusy/vedo/assets/71385431/59108356-1830-4b55-b01c-21086f78e311)

**Comment by marcomusy** [comment]
Yes you can create an Assembly:

```py
assem = myline + myobj
show(assem)
```

**Comment by OhmPuchiss** [comment]
I have combined both object into assembly. However, I have not get both object to move together. 
![image](https://github.com/marcomusy/vedo/assets/71385431/6e879746-d01b-4de3-bb66-a95e10f46901)
![image](https://github.com/marcomusy/vedo/assets/71385431/0fae8bff-9795-423c-8500-1b3cc5a3a19d)

**Comment by marcomusy** [comment]
Did you manage to do it?

**Comment by OhmPuchiss** [comment]
Since vedo is made on VTK, Can you create a python code to make this function ?
![image](https://github.com/marcomusy/vedo/assets/71385431/86baaf46-b084-45e1-88a3-cb2445385f75)

**Comment by OhmPuchiss** [comment]
I am not good at coding yet and I believe you could help me to create something like this in vedo.

**Comment by marcomusy** [comment]
You can create a picture-in-picture rendering with any sort of 3d object or axes. 
This is a good example to start with
https://github.com/marcomusy/vedo/blob/master/examples/other/inset.py
https://github.com/marcomusy/vedo/tree/master/examples/other/icon.py
https://github.com/marcomusy/vedo/tree/master/examples/volumetric/slicer1.py

**Comment by OhmPuchiss** [comment]
thank you for your assistance. I have one question. When we apply the transformation, the mesh.vertices still have the ols data from the dataset. How do we get the new coordinate information after transformation ?.

**Comment by OhmPuchiss** [comment]
*old

**Comment by marcomusy** [comment]
No, `mesh.vertices ` will hold the new coordinates after `.apply_transform(T)`

**Comment by OhmPuchiss** [comment]
I think I combine the mesh and other mesh into assembly and when I apply transformation matrix. The mesh.vertices inside the assembly data still hold the old data.
